src/components/lab/Lattice.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import LatticeRow from './LatticeRow';
+import { NUMBER_OF_ROWS_IN_LATTICE } from '../../config/constants';
+
+const Lattice = ({ stageDimensions }) => {
+  // create an array to map onto LatticeRow components
+  const rows =  [...Array(NUMBER_OF_ROWS_IN_LATTICE).keys()]
+
+  return rows.map((row) => (
```

Comment:
I guess you could go with using the index from `map`instead of creating the array above?

Reply:
not sure I understand. can you write the code snippet you are suggesting?